<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWindow</class>
 <widget class="QWidget" name="LogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Log</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="levelComboBox">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <item>
        <property name="text">
         <string>0 - ERROR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 - WARN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 - INFO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 - DEBUG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ALL</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
